.travis.yml

```diff
@@ -198,12 +198,16 @@ install:
   - conda install -c conda-forge --no-update-deps --no-deps pyshp || pip install pyshp
   # install packages not available via conda
   - pip install geographiclib
+  
+  - pip install numpy
+  
   # current pyimgur stable release has a py3 incompatibility
   - pip install https://github.com/megies/PyImgur/archive/py3.zip
   - pip freeze
   # XXX quick and dirty workaround for problems with pip wanting to upgrade certifi
   - if [[ "${PYTHON_VERSION}" == '3.5' ]]; then
-      conda remove --offline certifi;
+      #conda remove --offline certifi;
```

Comment:
If you don't need `certifi`, then you should just remove this entire block.

Reply:
It gets pulled in anyway and the pip installs at the end after conda installs can fail when dependency version resolution fails because pip refuses to uninstall conda-installed packages. But like I stated above, this is maintenance 1.2.x only, so very low priority for me. That being said if any experienced obspy maintainer feels a strong need to focus on the 1.2.x maintenance line, we could have somebody head that part

Reply:
@QuLogic the entire problem is that ```certifi``` gets uninstalled with uninstalls a bunch of other modules causing everyone's builds to fail.

Reply:
@zurgeg ah my bad, i didn't double check and thought `--offline` already implied leaving dependencies untouched. So I actually wanted to do `conda remove --offline --force certifi` to only remove certifi itself, so that pip can bring in the desired version it wanted in a subsequent `pip install` step. Anyway, like I said this issue is gone in `master` anyway, with Py35 dropped.

Reply:
@megies shouldn't we just drop support for Python 3.5 in 1.2.x maintenance then? That seems like the best idea. Another alternative that we could do is drop 1.2.x and make a 2.0 with 3.5 dropped.